src/modules/flow/iio/iio.json

```diff
@@ -308,6 +368,21 @@
          "data_type": "float",
          "description": "Temperature read, in milli degrees Celsius",
          "name": "OUT"
+        },
+        {
+         "data_type": "float",
+         "description": "Scale to applied to device raw readings",
+         "name": "SCALE"
+        },
+        {
+         "data_type": "float",
+         "description": "Offset to be added to device raw readings",
+        "name": "OFFSET"
+        },
+        {
+         "data_type": "int",
+         "description": "Sampling frequency of the sensor. If -1, use device default",
+         "name": "SAMPLING_FREQ"
```

Comment:
since we are using frequency everywhere, I guess it would be better `SAMPLIGING_FREQUENCY` don't you think ?